### PR TITLE
s/gpr_ref/grpc_core::RefCount/ for frequent users.

### PR DESCRIPTION
**Pull request by arjunroy**

### CODE REVIEW

File: src/core/lib/iomgr/ev_epollex_linux.cc [hunk 1]
```diff
@@ -584,30 +585,26 @@ static grpc_error* pollable_create(pollable_type type, pollable** p) {
 
 #ifdef NDEBUG
 static pollable* pollable_ref(pollable* p) {
+  const char* reason = nullptr;
+  grpc_core::DebugLocation debug_location;
 #else
 static pollable* pollable_ref(pollable* p, int line, const char* reason) {
```
Comment (soheilhy) [inline src/core/lib/iomgr/ev_epollex_linux.cc]: I would capture `grpc_core::DebugLocation` here and use `DEBUG_LOCATION` when calling `pollable_ref`.

**Comment by soheilhy** [comment]
Thank you for converting to `RefCount`!

File: src/core/lib/iomgr/ev_epollex_linux.cc [hunk 2]
```diff
@@ -135,17 +136,13 @@ static char* pollable_desc(pollable* p) {
...
-static void pollable_unref(pollable* p);
-#define POLLABLE_REF(p, r) pollable_ref(p)
-#define POLLABLE_UNREF(p, r) pollable_unref(p)
-#else
-static pollable* pollable_ref(pollable* p, int line, const char* reason);
-static void pollable_unref(pollable* p, int line, const char* reason);
-#define POLLABLE_REF(p, r) pollable_ref((p), __LINE__, (r))
-#define POLLABLE_UNREF(p, r) pollable_unref((p), __LINE__, (r))
-#endif
+static pollable* pollable_ref(pollable* p,
```
Comment (soheilhy) [inline src/core/lib/iomgr/ev_epollex_linux.cc]: nit: do you need the forward decl here? would you be able to move them (and the macros) down next to the function?

Comment (arjunroy) [reply src/core/lib/iomgr/ev_epollex_linux.cc]: moved up instead

File: src/core/lib/iomgr/ev_epollex_linux.cc (hunk 1 above)
Comment (arjunroy) [reply src/core/lib/iomgr/ev_epollex_linux.cc]: done

**Pull request merged**